app/controllers/sign_up/select_email_controller.rb

```diff
@@ -54,6 +55,10 @@ def last_email
       end
     end
 
+    def verify_multiple_emails
+      redirect_to sign_up_completed_path if user_emails.count < 2
+    end
+
```

Comment:
I noticed you can theoretically get to this page with only 1 email and it doesn't make sense unless you have multiples.

Reply:
There was some recent discussion about whether we wanted to consolidate to always show "Change" link / screen regardless of how many email addresses a user has, so this might change.

If keep it, can you add some test coverage for the expected behavior when visiting this screen with a single email address?

Reply:
Added with [a74a86d](https://github.com/18F/identity-idp/pull/11439/commits/a74a86d665794da0b8c5922f94ac5ff959bfda38)